views/display.html

```diff
@@ -14,6 +14,37 @@
             </div>
         </div>
 
+        <div id="users">
```

Comment:
I would suggest renaming the existing `.user` css class to just `.card` and using the same one here so that all the css responsiveness works for both, since they're all cards anyway basically